Firestore/Example/Tests/Model/FSTFieldValueTests.m

```diff
@@ -26,6 +26,7 @@
 #import "Firestore/Source/Model/FSTFieldValue.h"
 #import "Firestore/Source/Model/FSTPath.h"
 
+#import "Firestore/Example/Tests/API/FSTAPIHelpers.h"
```

Comment:
Hrm...  I didn't realize that these tests would end up relying on FSTAPIHelpers.h.  That makes it less obvious that my suggested separation makes sense. :-/

I think the difficulty comes from the fact that FIRGeoPoint is used as both a "model" class and an "API" class. I'd suggest moving FSTTestGeoPoint back into FSTHelpers.h/.m and adding a comment something like:

// Note that FIRGeoPoint is a model class in addition to an API class, so we put this helper here instead of FSTAPIHelpers.h

Reply:
done